### ISSUE TITLE
Chat answer layout differs in oai from deployment

### ISSUE THREAD

**Issue by RobSch1406** [Azure-Samples/azure-search-openai-demo#1584]
### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [x ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

Hello together, 

I am trying to ease up the use of our own deployment. Basically I try to have the same interpretation of the answer using the api then using oai.

I am missing the copy function of a generated code and that the code is beeing highlighted in a seperate frame. 

basically the markdown function differs. Any possibility to solve this?

How to change the result layout? 

Thanks and best regards

**Comment by RobSch1406** [comment]
got it fixed using correct markdown features in the answer.tsx file
